docs/user/communication.rst

```diff
@@ -41,99 +45,196 @@ Exemplary exercise detail page with posts:
 |exercise-posts|
 
 Course-wide Posts & Course Communication Overview
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------
 
-Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as ``Organization`` or ``Tech Support``.
-These posts can only be created on the course communication overview, which is shown in the screenshot below.
+Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as
+``Organization`` or ``Tech Support``. These posts can only be created on the course communication overview, which is shown
+in the screenshot below.
 
 The ``Communication`` space of an Artemis course serves as overview for *all* posts in a course.
 Hence, course-wide posts as well as exercise posts and lecture posts are listed.
 Here, users can easily query, sort, and filter existing posts.
 
 |course-posts|
 
+Secluded Communication
+----------------------
+Users of a course can communicate in private via the Messages page. (see image below) The page consists of a collapsible
+Conversation sidebar on the left, where users can search for other participants of the current course and start a conversation
+with them.
+
+If the recipient is browsing another conversation when they receive a new message, an envelope icon appears in their
+Conversation Sidebar, next to the affiliated user who has sent the message. This way, users become aware of the new message
```

Comment:
```suggestion
Conversation sidebar, next to the affiliated user who has sent the message. This way, users become aware of the new message
```
Having the titles as they appear in the Artemis UI capitalised looks good. Sidebar isn’t a part of that, though. (also consistency with line 63)

Reply:
Agreed 👍🏻